ipc/cli/src/commands/checkpoint/relayer.rs

```diff
@@ -68,6 +70,25 @@ impl CommandLineHandler for BottomUpRelayer {
                 .checkpoint_interval_sec
                 .unwrap_or(DEFAULT_POLLING_INTERVAL),
         );
+
+        if arguments.enable_metrics {
+            // Register ipc metrics.
+            let metrics_layer = ipc_metrics::layer();
+            let tracing_subscriber_registry = tracing_subscriber::registry().with(metrics_layer);
+            tracing::subscriber::set_global_default(tracing_subscriber_registry)
+                .expect("Unable to set a global collector");
+
+            let prometheus_registry = prometheus::Registry::new();
+            ipc_metrics::register_app_metrics(&prometheus_registry)
+                .context("failed to register metrics")?;
+
+            // Start metrics export HTTP server.
+            let addr: SocketAddr = arguments.metric_export_bind_address.parse().unwrap();
```

Comment:
```suggestion
            let addr: SocketAddr = arguments.metric_export_bind_address.parse().expect("failed to parse metrics bind address");
```

Almost never a good idea to just panic without any context.